addons/kotsadm/alpha/install.sh

```diff
@@ -186,6 +182,25 @@ function kotsadm_api_patch_prometheus() {
     cp "$DIR/addons/kotsadm/alpha/patches/api-prometheus.yaml" "$DIR/kustomize/kotsadm/api-prometheus.yaml"
 }
 
+function kotsadm_metadata_configmap() {
+    local src="$1"
+    local dst="$2"
+
+    # Create an empty application-metadata.yaml to keep kustomization happy
+    # There's a case where a slug is not specified, and thus no application metadata.
+    # In that case we won't create the ConfigMap for the metadata.
+    touch $dst/kotsadm-application-metadata.yaml
+
+    if [ "$AIRGAP" != "1" ] && [ -n "$KOTSADM_APPLICATION_SLUG" ]; then
+        # If slug exists, but there's no branding, then replicated.app will return nothing.
+        # (application.yaml will remain empty)
+        echo "Retrieving app metadata: url=$REPLICATED_APP_URL, slug=$KOTSADM_APPLICATION_SLUG"
+        curl $REPLICATED_APP_URL/metadata/$KOTSADM_APPLICATION_SLUG > "$src/application.yaml"
+        cp "$src/application.yaml" "$dst/"
+        kubectl create configmap kotsadm-application-metadata --from-file="$dst/application.yaml" --dry-run -oyaml > "$dst/kotsadm-application-metadata.yaml"
```

Comment:
On airgap installs this file will be included already at "$src/application.yaml" so 199 and 200 should be in another block guarded by a check for the file existing.

Reply:
The idea is to overwrite the file.  Although I'll go back and retest with airgap.

Reply:
Ok, I addressed this in my latest commits.  Although I'm finding issues when building airgap installer, so unable to find a good way to test this path.